packages/docs-reanimated/docs/debugging/accurate-call-stacks.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: accurate-call-stacks
+sidebar_label: Accurate Call Stacks
+sidebar_position: 1
+---
+
+# Accurate Call Stacks
+
+When debugging Reanimated code, you may encounter error or warning call stacks that do not clearly indicate the root cause of the problem. These stacks can be misleading, as they often highlight code from Reanimated's internals rather than the misuse of the Reanimated API that is the source of the problem.
+
+To address this, Reanimated provides a Metro configuration wrapper called `wrapWithReanimatedMetroConfig`. This wrapper automatically adjusts your Metro config to enhance the accuracy of call stacks in warnings and errors generated by the Reanimated library.
```

Comment:
```suggestion
To address this, Reanimated provides a Metro configuration wrapper called `wrapWithReanimatedMetroConfig`. This wrapper automatically adjusts your Metro config to improves the accuracy of call stacks in warnings and errors generated by the Reanimated library.
```
i detest the word `enhance` since the day the chatgpt was born

Reply:
GOAT

Reply:
Well, I like to use the `enhance` word, but can change it to `improve` if you prefer.